lib/LruMemoize.js

```diff
@@ -27,17 +27,19 @@ export default class LruMemoize {
    * @param {string} options.key - A key for the cache.
    * @param {Function<Promise>} options.fn - A Function that returns a
    *   promise to memoize.
+   * @param {object} [options.args] - The arguments to pass into the function
+   *   `fn`.
    *
    * @returns {Promise} - The result of the memoized promise.
   */
-  async memoize({key, fn}) {
+  async memoize({key, fn, args}) {
```

Comment:
the current design is that `fn` would be a closure which is how its currently being used... for example
```
const arg1 = 'foo';

const fn = () => someFunction({arg1});

await lruMemoize.memoize({key: 'myKey', fn})
```

Reply:
we should probably provide that as `@example` so future devs see it.